features/networking/sdn.feature

```diff
@@ -300,3 +300,34 @@ Feature: SDN related networking scenarios
     Then the step should succeed
     And the output should contain "Hello OpenShift"
     """
+    
+  # @author anusaxen@redhat.com
+  # @case_id OCP-25787
+  @admin
+  Scenario: Don't write CNI configuration file until ovn-controller has done at least one iteration
+    Given the env is using "OVNKubernetes" networkType
+    And I select a random node's host
+    #Checking controller iteration 1
+    When I run command on the "<%= node.name %>" node's sdn pod:
+      | bash | -c | ovs-appctl -t ovn-controller connection-status |
```

Comment:
should be `ovn-appctl` instead of `ovs-appctl`

Reply:
Yea, the ovn 2.12. Thanks for catching that. Updated whole case and re-ran to make sure http://pastebin.test.redhat.com/843959